src/composer.lock

```diff
@@ -1,10 +1,10 @@
 {
     "_readme": [
         "This file locks the dependencies of your project to a known state",
-        "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
+        "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "790e0f54fbc4dd9664d227e9ef479bd4",
+    "content-hash": "6e9e7c3b261255bdd2ccbc9edfd403ba",
```

Comment:
Please update `src/composer.lock` (https://travis-ci.org/fossology/fossology/jobs/597459214#L721)

Reply:
Updated changes in the latest commit

Reply:
I am sorry, which commit you are referring to as I can not see any new commit since the comment?

Reply:
I have pushed my changes to feat/spasht_5.0